questionnaires/covid19_symptoms/covid19_symptoms_armt.json

```diff
@@ -0,0 +1,866 @@
+[
+    {
+        "field_name": "cough_wk",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "yesno",
+        "field_label": "Have you had a cough in the last two weeks",
+        "select_choices_or_calculations": "",
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "cough_rate",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "Rate how severe your cough has been",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 None "
+            },
+            {
+                "code": "1",
+                "label": "1 Mild "
+            },
+            {
+                "code": "2",
+                "label": "2 Moderate "
+            },
+            {
+                "code": "3",
+                "label": "3 Severe"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[cough_wk] = '1'",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": "responses['cough_wk'] ==  '1'"
+    },
+    {
+        "field_name": "covid19_mucus",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "checkbox",
```

Comment:
@afolarin I think this should be radio? (I can make the changes in REDCap)

Reply:
I wonder if it should be split as this item is really 2 questions rolled in one. Let me tak a look tonight